core/frontend/src/extension/providers/RemoteExtensionProvider.ts

```diff
@@ -47,14 +47,12 @@ export class RemoteExtensionProvider implements ExtensionProvider {
    * Throws an error if the provided manifestUrl is not accessible.
    */
   public async getManifest(): Promise<ExtensionManifest> {
-    const options: RequestOptions = { method: "GET" };
-    const response = await request(this._props.manifestUrl, options);
-    const data = response.body || (() => {
-      if (!response.text)
-        throw new Error("Manifest file was empty.");
-      return JSON.parse(response.text);
-    })();
-    return data;
+    const response = await request(this._props.manifestUrl, "text");
```

Comment:
wonder for this one should be "json" not "text", then can avoid having to parse it later

Reply:
I believe `superagent` used to default to `"text"` if `responseType` was not provided.

Changed to `"json"` anyway, because end result should be the same.